src/PageLayout/PageLayout.tsx

```diff
@@ -466,6 +466,25 @@ Content.displayName = 'PageLayout.Content'
 // ----------------------------------------------------------------------------
 // PageLayout.Pane
 
+type Measurement = `${number}px`
```

Comment:
Is it possible that teams may want to include other units like `%` or `rem` or is `px` kind of what's expected here?

Reply:
I could see that possibility. One of my concerns regarding having more options is it would complicate converting to numbers. The `updatePaneWidth` helper currently expects a number. I don't know a great way to handle that going from percentages, rems, ems, etc.